rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/70_adjacency_matrix.yml

```diff
@@ -125,3 +125,40 @@ setup:
 
   - match: { aggregations.conns.buckets.3.doc_count: 1 }
   - match: { aggregations.conns.buckets.3.key: "4" }
+
+
+---
+"Show only intersections":
+  - skip:
+      version: " - 2.13.99"
+      reason: "show_only_intersecting was added in 2.14"
+      features: node_selector
+  - do:
+      node_selector:
+        version: "2.14.0 - "
+      search:
```

Comment:
Related to my comment below about the serialization/deserialization, you'll need to initially set this check to skip `version: " - 2.99.99"`. Only after the change has been backported to 2.x can we update it to point to the correct version (likely 2.18 at this point).